.github/workflows/markdown.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Please remove these comments. If we think that a template is necessary we can add one.

Reply:
Sure.